content/docs/user-guide/managing-external-data.md

```diff
@@ -33,25 +29,26 @@ Currently, the following types (protocols) of external outputs (and
 - HDFS
 - Local files and directories outside the <abbr>workspace</abbr>
 
-> Note that these are a subset of the remote storage types supported by
-> `dvc remote`.
+> Note [remote storage](/doc/command-reference/remote) is a separate feature,
+> and that external outputs are not pushed or pulled from/to DVC remotes.
 
-> Avoid using the same [DVC remote](/doc/command-reference/remote) (used for
-> `dvc push`, `dvc pull`, etc.) for external outputs, because it may cause file
-> hash overlaps: the hash of an external output could collide with a hash
-> generated locally for another file with different content.
```

Comment:
Note that I totally removed this note. How likely is this scenario really? And if we're worried about it, should DVC core handle it as an error, at least?

Reply:
It's hard for DVC to handle this. It will change a bit in DVC 2.0, so it might be safe to remove it. On the other hand, it's a nature of clouds- they generate some hash that might differ from the one we generate with DVC. And there is no any guarantees. The best we can do is to have per cloud instruction- is it safe or not mixing those things.

Reply:
I would keep it for now. Don't see a good reason to remove it.

Reply:
> Don't see a good reason to remove it.

Doing so per https://github.com/iterative/dvc.org/pull/1945#pullrequestreview-531982134 and https://github.com/iterative/dvc.org/pull/1945#pullrequestreview-531992838 above. And since it seems to me like an extremely rare edge case. And if it will change in 2.0 we can think about what to say about it then.

Reply:
> It's hard for DVC to handle this.

BTW if the problem is "hash overlaps: the hash of an external output could collide with a hash generated locally for another file" that sounds like a very precise condition that could be caught/handled. But I'm not sure.

Reply:
@jorgeorpinel There is no guarantee it will go away in 2.0. This note discourages people from misusing our incomplete ext feature, so better to not remove it for now.

Reply:
OK sure, I rolled this back (mostly) in defd99d8 @efiop.

What did you mean by "incomplete ext feature" thouogh? ext = external data? Just curious